assets/queries/dockerfile/copy_instead_of_add/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Copy_instead_of_add",
+  "queryName": "Copy Instead of Add",
+  "severity": "LOW",
+  "category": "Image",
+  "descriptionText": "Should use COPY instead of ADD unless, running a tar file",
+  "descriptionUrl": "https://docs.docker.com/engine/reference/builder/#add"
+}
```

Comment:
This belongs to another query, you should rebase your branch with master and amend your commit to remove this query from this branch